.babelrc

```diff
@@ -1,14 +1,21 @@
 {
```

Comment:
updated to allow the materials designer to run locally. Changes from https://github.com/Exabyte-io/materials-designer/pull/25/files